### DISCUSSION TITLE
After the update, there are a lot of people I don't know and don't follow, and I can't close them individually

### DISCUSSION THREAD

**Discussion by DreamOfIce** [community/community#37421] [Other Feature Feedback, Questions, & Ideas]
This prevents me from getting the information I want as much as I used to: 
- what people I follow are up to,
- what people are following me,
- what new stars my repository is getting.
 They are all mixed in with a lot of dynamics from people I don't know at all, even if I turn off " Recommendations".
![7E6A078A-A645-48EF-8F41-6BD534AFD567](https://user-images.githubusercontent.com/80821706/198181752-84ce2ed3-fbf4-4ca4-806d-2a64cc802e50.jpeg)
The picture above is my GitHub home page, and the top few entries are all from people I don't follow or even have no contact with at all. This is with only star checked, by default even every commit from many strangers is listed.
By my calculations, the percentage of information I want is less than 20%, and I can't filter the information I want from it at all!

**Comment by tomaarsen** [comment]
> * what people I follow are up to,
> * what people are following me,
> * what new stars my repository is getting.

That is also exactly the information I desire, and this feed update has made it nearly impossible to get that information. 
I very much hope that GitHub splitst this back into "Feed" with the old information, and the "Explore" tab with this type of social media style spam.

**Comment by ossanna16** [comment]
Hi there @DreamOfIce and welcome to our community! Thank you for the great feedback 🙂
